src/components/shared/editable-detail/editable-detail.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import React, { useContext } from 'react';
```

Comment:
```suggestion
import { useContext } from 'react';
```
with react 17, you shouldn't need to import the react itself